action.yml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: 'openjdk Repo'
     required: false
     default: 'openjdk-tests:master'
+  openj9_repo:
+    description: 'openj9 Repo'
+    required: false
+    default: 'openj9:master'
```

Comment:
Should use the new repo? i.e., `eclipse-openj9:openj9`

Reply:
Talked with @renfeiw , this is used as a check. It is not used for constructing openj9 git repo url, so it does not need to be changed.

Reply:
openj9 is the OpenJ9 main code base under eclipse-openj9 : https://github.com/eclipse-openj9/openj9. I think openj9 is a correct one. Also I believe `openj9_repo` is set as the same way as two other repos do, which only use default as a flag. We actually don't need the default value as we can check if the parameter is empty or not. I will open a separate issue to do that.